### DISCUSSION TITLE
In RE 1.6, what variable makes damage reduce at a distance even if you hit a perfect shot?

### DISCUSSION THREAD

**Discussion by ghost** [redeclipse/base#1190] [Help and Support]
And how does it work? I tried taper and the gun still did full damage at max range.

**Comment by qreeves** [comment]
`taper` set to values of 1 to 4 works across the lifetime of the projectile, and 5 to 6 works across distances. You are likely looking to set `taper` to `2` and `taperout` to `1.0` which will make the projectile taper out to nothing as it reaches the end of its lifetime.
